### PR TITLE
add bug report and feature request templates #1

### PR DESCRIPTION
**Pull request by mahdiatallah**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```
Comment (AdrienDeschenaux) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]: I'm not sure to understand what is meant by problem here. If I were to fill this feature request I could be a bit confused. Maybe we could formulate this as : 
- `Is there a specific issue your feature aims to address ?` 

or maybe 

- `Why would your feature be an improvement of the current app?`.

Let me know what you think of my propositions

**Review by AdrienDeschenaux — COMMENTED** [review]
Those templates are good ! 
I have the feeling that a small section could be reformulated let me know what is your opinion about my proposition

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (mahdiatallah) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: ´Is there a specific issue your feature aims to address ?´ sounds good!

Comment (userdarius) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Yup much better!

**Comment by userdarius** [comment]
Yup all seems good here I agree with @AdrienDeschenaux 's message and the templates are great!

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (AdrienDeschenaux) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Great thank you for making this change, I'll merge this on main !

**Review by userdarius — APPROVED** [review]
All good on my end the templates are concise and detailed enough for a good bug report and feature request!

**Review by AdrienDeschenaux — APPROVED** [review]
Thank you for the change, this is all good now

**Pull request merged**